### ISSUE TITLE
Issue: Problems with unresponsive buttons on curved edge screens.

### ISSUE THREAD

**Issue by chrismaliszewski** [NPBruce/valkyrie#1126]
I'm testing v2.3.02b on S8+ but I presume it's a general issue. I find some buttons highly unresponsive. I can tap them several times and nothing may happen until "the next" tap. It's quite frustrating. They have one thing in common: they are at the bottom of a screen.

1. The most annoying, buttons during playing a scenario at the bottom. **Especially frustrating** is Next Phase as it's used so many times.
2. Back buttons at the bottom of a screen. 

Is it just me?

**Comment by scrubbless** [comment]
1. The splash screens in the normal game help with this. Believe it or not in 1.X that delay was up to 15 seconds. That specific issue was due to the save game function, that may be a good place to start.

2. I think this is the s8+ edge more than Valkyrie.

I have an S7edge and S9+ both have that issue. My old HTC one m8 and lg g5 are both fine.

**Comment by chrismaliszewski** [comment]
Regarding 1., 15s wow, that's crazy :). 
Regarding 2., I thought so. I mean, that it's a curved-edge screens' problem. Thanks, @scrubbless, for your comment!

After your reply, I made a small investigation. I found this article: https://www.samsung.com/in/support/mobile-devices/how-to-prevent-edge-screen-touch-issue/. I installed Edge Touch app and played with the edges. 
The default sensitive-edge-area size in my case was:
![Screenshot_20190324-110326_Valkyrie](https://user-images.githubusercontent.com/7532602/54877900-1daacc80-4e25-11e9-8688-d8b3eb4a2091.jpg)
so I made it smaller:
![Screenshot_20190324-110250_Valkyrie](https://user-images.githubusercontent.com/7532602/54877905-27cccb00-4e25-11e9-9eed-5ece753afdca.jpg)
and everything is much more responsive now. 

The above solution works fine for Samsung phones. 
I'm closing the issue as it's nt related to Valkyrie and it's an invidual's phone issue.
